parse/test/bugreports/generic_new_and_make_slice.go

```diff
@@ -0,0 +1,15 @@
+package bugreports
```

Comment:
This pkg is a good idea. I think it would be nice if it was specified as a comment how test cases relate to github issues.

Reply:
Added a brief comment to this file.